07-documentation.md

```diff
@@ -0,0 +1,80 @@
+# REST documentation
+
+With the separation of frontend and backend it has become of critical importance that the REST interface is clearly documented. It is possible that frontend and backend tasks are performed by different developers, which simply means that the frontender should know what requests to call and what output to expect.
+
+Besides an overview of resources the documentation should also discuss cross cutting concerns, such as status codes, error handling and security. The REST document serves as contract, helping consumers of the interface and ensuring that the interface stays consistent and clear.
+
+Writing REST documention takes time, but should be worth it in the process of a project. REST documentation minimizes communication issues between developers and helps new developers become productive in a shorter time span. To really benefit from a REST document it has to stay accurate. Outdated documentation can bring more problems than solutions.
+
+# Tooling
+
+We use tooling to write and maintain the REST documentation in a minimal amount of time. In the sections below two tools are described and compared. Use the tool that best fits your situation.
```

Comment:
I am not really found of using first person pov in a technical document. But what do you guys think @42BV/developer ?